backend/src/apiserver/client/subject_access_review.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2020 Arrikto Inc.  All Rights Reserved.
+
+package client
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	"k8s.io/client-go/rest"
+)
+
+func createSubjectAccessReviewClient() (v1.SubjectAccessReviewInterface, error) {
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to initialize kubernetes client.")
```

Comment:
When failing with in cluster config, can we try kubeconfig, so that we can run API server locally?

Reply:
If I understand correctly, this is the same with kubernetes core client initialization ([source](https://github.com/kubeflow/pipelines/blob/a8bd4682656e75b5c5cd48d75ffdf23e97bf99aa/backend/src/apiserver/client/kubernetes_core.go#L25-L36))

This means we should patch both places (and every other place) if we want to run it locally.
Would it be better to file an issue and have a distinct PR for it?

Reply:
Thanks, I wasn't aware of the existing convention, makes sense as separate issue and PR.

Reply:
I created an issue to track the effort: https://github.com/kubeflow/pipelines/issues/4738
So I'm marking this as resolved as well